rpcs3/rpcs3qt/gs_frame.cpp

```diff
@@ -8,6 +8,13 @@
 #include <QTimer>
 #include <QThread>
 
+#ifdef VK_USE_PLATFORM_WAYLAND_KHR
```

Comment:
This whole block from line 11 to 16 should be in an ifndef _WIN32 block. That should fix appveyor CI and allow this set to be mergeable